microprofile/security/src/main/java/io/helidon/microprofile/security/SecurityCdiExtension.java

```diff
@@ -15,17 +15,109 @@
  */
 package io.helidon.microprofile.security;
 
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.logging.Logger;
+
+import javax.annotation.Priority;
 import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.context.Initialized;
 import javax.enterprise.event.Observes;
+import javax.enterprise.inject.spi.BeanManager;
 import javax.enterprise.inject.spi.BeforeBeanDiscovery;
 import javax.enterprise.inject.spi.Extension;
 
+import io.helidon.common.HelidonFeatures;
+import io.helidon.common.HelidonFlavor;
+import io.helidon.common.context.Contexts;
+import io.helidon.config.Config;
+import io.helidon.microprofile.cdi.RuntimeScoped;
+import io.helidon.microprofile.server.JaxRsCdiExtension;
+import io.helidon.microprofile.server.ServerCdiExtension;
+import io.helidon.security.AuthenticationResponse;
+import io.helidon.security.ProviderRequest;
+import io.helidon.security.Security;
+import io.helidon.security.integration.jersey.SecurityFeature;
+import io.helidon.security.integration.webserver.WebSecurity;
+import io.helidon.security.providers.abac.AbacProvider;
+
 /**
  * Extension to register bean {@link SecurityProducer}.
  */
+@Priority(10000)
```

Comment:
Hmm; I don't remember CDI honoring `@Priority` on extensions themselves.  Maybe it does.  I know it honors `@Priority` on observed event parameters.  Was that what you meant to do?

Reply:
I think I wrongly copied it from MpService. Will move to event params